docs/hybrid-slurm-cluster/hybrid-slurm.md

```diff
@@ -0,0 +1,423 @@
+* `/opt/apps`
+  * Required: No
+  * Contains shared applications useful for running slurm jobs, such as open
+    MPI.
+
+By default, the compute node will fail to startup if any of these directories
+fail to mount from the controller. For more information, see the
+[Prepare NFS](#prepare-nfs) section.
+
+### Slurm versions
+
+The [slurm-gcp hybrid module][hybridmodule] only supports versions 21 and 22 of
+Slurm on the cloud compute nodes. In addition, since the controller must be in
+sync with Slurm version installed in the cloud compute nodes, the controller
+must be upgraded to version 21 at minimum before making use of the
+[hybrid module][hybridmodule].
+
+The default Slurm cloud compute node images install version 22.05.3. If your
+controller is running with slurm version 21, you can create a custom image with
+your desired slurm version. For more information, see the section titled
+[Creating a Slurm Compute Image](#creating-a-slurm-compute-image).
+
+### User and Group IDs
+
+The default [slurm-gcp] cloud compute VM disk images have the following user
+and group IDs for the slurm and munge users and groups:
+
+* `slurm`
+  * User ID: 981
+  * Group ID: 981
+* `munge`
+  * User ID: 980
+  * Group ID: 980
+
+If your Slurm controller sets different user and group IDs for these users, a
+custome cloud compute VM disk image must be created with these values updated.
+For more information and instructions, see
+[Creating a Slurm Compute Image](#creating-a-slurm-compute-image).
+
+### Power Saving Operations
+
+The hybrid partitions rely on the Slurm [power saving][powersaving]
+functionality for creation and tear down of the cloud compute VM instances. If
+this functionality is also being used on your local slurm cluster, the
+[slurm hybrid module][hybridmodule] is not be supported.
+
+[powersaving]: https://slurm.schedmd.com/power_save.html
+
+## Troubleshooting
+
+For troubleshooting suggestions, visit [troubleshooting.md](./troubleshooting.md).
+
+## Before You Begin
+
+### Cloud Environment Setup
+
+#### Select or Create a GCP Project
+
+This process will require a GCP project where the cloud partitions will create
+new compute VM instances to complete slurm jobs.
+
+Identify or create the project you intend to use. For more information, visit
+the Google Cloud documentation on
+[Creating and Managing Projects](https://cloud.google.com/resource-manager/docs/creating-managing-projects)
+
+#### Enable Required APIs
+
+At minimum, the following APIs are required to complete these instructions:
+
+* [Compute Engine API][computeapi]
+* [Filestore API][fileapi]
+
+[computeapi]: https://cloud.google.com/compute/docs/reference/rest/v1
+[fileapi]: https://cloud.google.com/filestore/docs/reference/rest
+
+#### Set IAM Roles
+The authenticated service account used by the slurm controller must have the
+Editor role in the cloud burst project. For more information on authenticating
```

Comment:
There is some inconsistent terminology. `cloud project` vs `cloud burst project` vs `GCP project`. Let's pick one.

Reply:
I went with Google Cloud project where ever it made sense, and just project when it was clear, or overly verbose to use Google Cloud.